src/features/room/chat/ui/ReportChatMessageModal.module.css

```
@@ -0,0 +1,127 @@
+  height: 16px;
+  accent-color: #2f86ed;
+}
+
+.visuallyHidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)